postgres/9.6/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM postgres:9.6
+
+MAINTAINER Debezium Community
+
+# Install the packages which will be required to get everything to compile
+RUN apt-get update && apt-get install -y build-essential pkg-config git postgresql-server-dev-9.6 software-properties-common
```

Comment:
A [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/) is to place each command in `RUN` on a separate line. Another is to place each installed dependency on a separate line. Both make it much easier to read complex run statements.